certified-connectors/Kanban Tool/apiDefinition.swagger.json

```diff
@@ -0,0 +1,1728 @@
+        "description": "This operation update a task.",
+        "summary": "Update a task",
+        "operationId": "UpdateTask",
+        "consumes": [
+          "application/json"
+        ],
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "$ref": "#/parameters/board_id"
+          },
+          {
+            "$ref": "#/parameters/task_id"
+          },
+          {
+            "name": "task",
+            "in": "body",
+            "required": true,
+            "description": "Task details.",
+            "x-ms-summary": "Task details",
+            "schema": {
+              "$ref": "#/definitions/TaskUpdateRequest"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "success",
+            "schema": {
+              "$ref": "#/definitions/GetTaskResponse"
+            }
+          }
+        },
+        "x-ms-visibility": "important"
+      }
+    },
+    "/api/v1/boards/{board_id}/tasks/{task_id}/move.json": {
+      "put": {
+        "description": "This operation moves a task.",
+        "summary": "Move a task",
+        "operationId": "MoveTask",
+        "consumes": [
+          "application/json"
+        ],
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "$ref": "#/parameters/board_id"
+          },
+          {
+            "$ref": "#/parameters/task_id"
+          },
+          {
+            "name": "task",
+            "in": "body",
+            "required": true,
+            "description": "Task details.",
+            "x-ms-summary": "Task details",
+            "schema": {
+              "$ref": "#/definitions/TaskMoveRequest"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "success",
+            "schema": {
+              "$ref": "#/definitions/GetTaskResponse"
+            }
+          }
+        },
+        "x-ms-visibility": "important"
+      }
+    },
+    "/api/v1/boards/{board_id}/tasks/{task_id}/archive.json": {
+      "put": {
+        "description": "This operation archives a task.",
+        "summary": "Archive a task",
+        "operationId": "ArchiveTask",
+        "consumes": [
+          "application/json"
+        ],
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "$ref": "#/parameters/board_id"
+          },
+          {
+            "$ref": "#/parameters/task_id"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "success",
+            "schema": {
+              "$ref": "#/definitions/GetTaskResponse"
+            }
+          }
+        }
+      }
+    },
+    "/api/v1/boards/{board_id}/changelog.json": {
+      "get": {
+        "description": "This operation get a board activities.",
+        "summary": "Board activities",
```

Comment:
@spoterala Can you please add "Get " to the summary value as prefix? like below:
`"summary": "Get Board activities"`